app.json

```diff
@@ -1,7 +1,7 @@
 {
   "scripts": {
     "dokku": {
-      "predeploy": "python manage.py compilescss && python manage.py collectstatic --noinput -i *.scss",
+      "predeploy": "python manage.py migrate && python manage.py compilescss && python manage.py collectstatic --noinput -i *.scss",
```

Comment:
We run migrations in postdeploy in all our other apps. I think we should aim for consistency unless there's a good argument to have this here?

Reply:
I've reverted. It was part of the deploy troubleshooting.